pkg/urbit/worker/main.c

```diff
@@ -24,6 +24,8 @@
 #include "all.h"
 #include <vere/vere.h>
 
+#define _CVX_ZUSE 309
```

Comment:
This particular naming convention is local to `vortex.c` -- caps 'cause it's a constant, _cv because it's static/local to `vortex.c`. Feel free to name this however you see fit!